### DISCUSSION TITLE
Stock filter list "Online Malicious URL Blocklist" not updated

### DISCUSSION THREAD

**Discussion by Procyon-b** [uBlockOrigin/uBlock-issues#2043] [General]
The version on the [server](https://curbengh.github.io/malware-filter/urlhaus-filter-online.txt) is still dated `Mon, 28 Feb 2022 00:14:36 +0000` 

It seems a little bit old. I don't remember seeing so many days between updates. (It's supposed to be `updated twice a day.`)
I've already posted a message on gitlab several days ago, but no reaction so far.

**Comment by gorhill** [comment]
I get `! Updated: Fri, 11 Mar 2022 00:14:59 +0000` on mine.

**Reply by Procyon-b** [reply]
I've tested on all my computers and they all download a file with this header: `! Updated: Mon, 28 Feb 2022 00:14:36 +0000`

**Reply by gorhill** [reply]
Ok, I get the out of date version when I force an update. However I get the latest version when I let auto update do the fetch. Auto update uses mirrors to avoid overloading the repo server. Manual update will cause the repo server to be used. I suggest you just mark the list as out of date by clicking the little clock, then leave it as is and let uBO auto update it in its next auto update cycle, it should then fetch from one of the mirrors.

**Reply by Procyon-b** [reply]
I've found the reason. I've the latest version on FF. But it's because it doesn't use the same url

`https://curben.gitlab.io/urlhaus-filter/urlhaus-filter-online.txt` in FF
`https://curben.gitlab.io/malware-filter/urlhaus-filter-online.txt` in all my chromes (edit: and the same url is also used in the [link here on their website](https://gitlab.com/curben/urlhaus-filter#url-based).)

**Reply by gorhill** [reply]
Hmm you're right, it seems even the mirrors are also broken.

**Reply by Procyon-b** [reply]
Of the 8 different urls (from their site), 2 are dead, 2 are up-to-date, 4 are outdated
